src/seam-os/client.ts

```diff
@@ -193,7 +193,7 @@ export class SeamOS {
     list: this._getGrab("/device_groups/list", "device_groups"),
     delete: this._curriedPost("/device_groups/delete"),
     update: this._curriedPost("/device_groups/update"),
-    add_device: this._curriedPost("/device_groups/add_device"),
+    // add_device: this._curriedPost("/device_groups/add_device"),
```

Comment:
this wasn't type checking because `/device_groups/add_device` isn't present in `routes.ts`, not sure how it got committed